app/models/feedback_session.rb

```diff
@@ -17,9 +17,11 @@ class FeedbackSession < ApplicationRecord
 #  session_date :date
 #  created_at   :datetime         not null
 #  updated_at   :datetime         not null
+#  comment_id   :bigint(8)
```

Comment:
está raro que la feedback session tenga una referencia al `comment_id`, no debería tenerla si es un `has_many`.
En general cuando teni una relación one-to-many pasa que el que tiene el has_many no tiene una referencia a los hijos en bdd, pero el que tiene el belongs_to sí.